server/server.js

```diff
@@ -23,11 +25,40 @@ app.get('/product/:id', (req, res) => {
 
 app.get('/data/grab', (req, res) => {
   dbConnection.connection.query('SELECT * FROM photos', (err, data) => {
-    if(err){return console.log(err, 'err')}
+    if(err){return cconsole.error(err, 'err')}
     res.send(data);
   })
 })
 
+app.post('/product/:id', (req, res) => {
+  dbConnection.connection.query('INSERT INTO xx_BLOB(IMAGE) VALUES(LOAD_FILE('someImage.jpg'))', (req, res) => {
```

Comment:
Your db.js file seems to have two query functions defined but you don't seem to be using them in your CRUD API? You might consider adding the queries you use here in your server file as functions to be exported from db.js